GeoHealthCheck/plugins/probe/ogc3dtiles.py

```diff
@@ -0,0 +1,68 @@
+from GeoHealthCheck.probe import Probe
+import requests
+
+
+class OGC3DTiles(Probe):
+    """
+    OGC3DTiles
+    """
+
+    NAME = 'GET Tileset.json and tile data'
+    DESCRIPTION = 'OGC3DTiles'
```

Comment:
PLease provide more extended, one line description. This text will appear in the Resource Edit UI, next to "Probe" in italics.